addons/microsoft_calendar/utils/microsoft_calendar.py

```diff
@@ -88,6 +91,13 @@ def delete(self, event_id, token=None, timeout=TIMEOUT):
                 raise e
             _logger.info("Microsoft event %s was already deleted" % event_id)
 
+    @requires_auth_token
+    def answer(self, event_id, answer, values, token=None, timeout=TIMEOUT):
+        url = "/v1.0/me/calendar/events/%s/%s" % (event_id, answer)
+        headers = {'Content-type': 'application/json', 'Authorization': 'Bearer %s' % token}
+        self.microsoft_service._do_request(url, json.dumps(values), headers, method='POST', timeout=timeout)
```

Comment:
No need to use `json.dumps(values, separators=(',', ':'))` instead of `json.dumps(values)` here ?

Reply:
Indeed I added the separators in google but it is not helpful as it is the default value.